### PR TITLE
juman 7.01 (new formula)

### PR DESCRIPTION
**Pull request by ymyzk**
Add JUMAN (a User-Extensible Morphological Analyzer for Japanese).

### CODE REVIEW

File: Library/Formula/juman.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  sha1 "e22eb113f40216b8a04d811484a94f93f834ae9a"
+
+  def install
+    system "./configure", "--disable-dependency-tracking",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system "juman", "-v"
```
Comment (mikemcquaid) [inline Library/Formula/juman.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

**Comment by ymyzk** [comment]
Thank you for reviewing.

Updated the test. A new test program executes `echo 京都大学 | juman | md5` and check its result.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by ymyzk** [comment]
:+1:

**Pull request closed without merging**